intranet/templates/eighth/admin/transfer_students.html

```diff
@@ -30,6 +30,16 @@
                 {% endif %}
             {% endcomment %}
         </form>
+    {% elif moved_students %}
+        <p>
+            The following students were already signed up for activities on {{ dest_act.block }}.
+            These signups have been replaced with {{ dest_act }}, and these students have been notified via email:
+        </p>
+        <ul>
+            {% for s in moved_students %}
+            <li>{{ s }}</li>
```

Comment:
I would want to print out the activity that each student was in. That way, if the admin who performs this action realizes this change messes something up, they can easily go through and revert the changes.

Reply:
This does. It prints the `EighthSignup` which includes the username, activity and block.

Reply:
Ah. Missed that.